core-addon/DataCollection.js

```diff
@@ -60,7 +85,7 @@ module.exports = class DataCollection {
     // Note that the schema namespace directly informs how data is segregated after ingestion.
     // If this is an enrollment ping for the pioneer program (in contrast to the enrollment to
     // a specific study), use a meta namespace.
-    return await this._sendEmptyPing("pioneer-enrollment", "pioneer-meta");
```

Comment:
Is the global enrollment still handled by ion (in about:ion)? I'm unclear what the relationship of this addon to the one that is fetched by about:ion.

Reply:
This addon is the Core Add-on described in [this architecture document](https://docs.google.com/document/d/1pebhTPg_1F4rGpdQYNbPa201MuDEBCHcIZqlbem8xXI/edit#). It will **replace** the current `about:ion` experience in Firefox.